Pipfile

```diff
@@ -25,7 +25,6 @@ python-redis-rate-limit = "*"
 python-json-logger = "*"
 lark-parser = "<0.7"
 astral = "==1.*"
```

Comment:
`lark-parser` and `astral` are subdependencies of `osm_humanized_opening_hours` that had to be pinned here. 
They should be removed now.